nestkernel/conn_builder.cpp

```diff
@@ -369,13 +371,29 @@ nest::ConnBuilder::change_connected_synaptic_elements( index sgid,
 void
 nest::ConnBuilder::connect()
 {
+  if ( symmetric_ && not supports_symmetric() )
+    throw NotImplemented(
+      "So far Connect doesn't support symmetric connections for this "
```

Comment:
I would remove the "So far". It implies a promise of coming attractions that we may not want to or be able to fulfill.